async.go

```diff
@@ -69,11 +68,9 @@ func processAsyncResults() {
 		}
 
 		// push to Core Data
-		event := &models.Event{Device: acv.DeviceName, Readings: readings}
-		ctx := context.WithValue(context.Background(), common.CorrelationHeader, uuid.New().String())
-		_, err := common.EventClient.Add(event, ctx)
-		if err != nil {
-			common.LoggingClient.Error(fmt.Sprintf("processAsyncResults - Failed to push event %v: %v", event, err))
-		}
+		cevent := models.Event{Device: device.Name, Readings: readings}
+		event := &ds_models.Event{Event: cevent}
+		event.Origin = time.Now().UnixNano() / int64(time.Millisecond)
+		common.SendEvent(event)
```

Comment:
@tobiasmo1 , thanks for the revision.  I thought we should add an error return from `common.SendEvent `and log something here if error occurs, but I will open another PR for this later.